### PR TITLE
Tidying up documentation and repo structure

### PR DESCRIPTION
**Pull request by joefroh**
I went ahead and started working through your instructions (thanks for that btw, super helpful).

I haven't had a chance to actually run this yet (I am a bit terrified that I will accidentally post something to /r/winnipegjets) but I think I solved one of your open items in the process of just walking through your instructions (tl;dr submodules are a way better way of bringing in a repo dependency than hard coding paths).

Feel free to give this a look over (and a test run). If it looks sane, take the request and I will keep cracking away at tidying things up.

### CODE REVIEW

**Comment by joefroh** [comment]
@KyleBrownCS ping

**Comment by KyleBrownCS** [comment]
I'll take a look this on Friday eve or Saturday. Got wiped by a cold these past few days

**Comment by joefroh** [comment]
Feel better!

**Comment by joefroh** [comment]
Hope you're feeling better and enjoyed that destruction of Ottawa last night :)

**Comment by KyleBrownCS** [comment]
Yup I'm all good/better now! I opened a review for the .gitmodules file with a question. I'm not sure if you can see it or not.

Everything looks good other than the one question I had! Let me know if you can't see the review, I might have done/configured something wrong

**Comment by joefroh** [comment]
I don't see the gitmodules thing... is it in this PR? you can just view the file and add a comment in line I believe.

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "gspread"]
```
Comment (KyleBrownCS) [inline .gitmodules]: Sorry about the delay here, I opened a review Saturday morning, I guess I didn't open/comment right? Anyways, here is a copy/paste of the contents.

I've never used .gitmodules before so my knowledge is basically this doc: https://git-scm.com/docs/gitmodules

This looks like a perfect solution to the import and revisioning problem!

I had an issue when I swapped to your PR branch where my gspread directory wasn't populated. After some google-fu I used git submodule update --init --force --remote which ended up working. Was this the right solution or approach to pull in the package? I'd like to add what ever step I missed (or this solution if it is correct) to the documentation, Everything else looks perfect and runs great.

Comment (joefroh) [reply .gitmodules]: I've added some lines to the readme doc to get the submodule pulled down. I honestly thought it was automatic... My bad. I've always been the one adding the module, not cloning it down so i've honestly never had to figure out the other side.

The instructions I added came from the doc you listed above and worked on a clean clone of my fork (forked my master, changed to my working branch, used the git commands listed). I think your one liner probably works to, but I generally avoid --force calls in git where possible. I think my commands are the canonical approach.

Comment (KyleBrownCS) [reply .gitmodules]: Awesome, thanks. Not sure what I did wrong the first time but it works on my local machine so I'm happy

**Pull request merged**